### PR TITLE
add histoprint recipe

### PR DESCRIPTION
**Pull request by mbhall88**
Addresses https://github.com/scikit-hep/histoprint/issues/15

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mbhall88** [comment]
@conda-forge-admin I am a first-time contributor so am unable to ping the relevant team (python) for this PR. It seems the windows build failed. I know nothing about windows so am unable to figure out how to rectify this.

File: recipes/histoprint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  number: 0
+  entry_points:
+    - histoprint=histoprint.cli:histoprint
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - click >=7.0.0
+    - numpy >=1.0.0
```
Comment (scopatz) [inline recipes/histoprint/meta.yaml:21]: You probably don't need numpy or click to build the package

File: recipes/histoprint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 841fd242eed078da8df5677b040785482a5edbf901f3c1fbcad20717ebcf7219
+
+build:
+  number: 0
+  entry_points:
+    - histoprint=histoprint.cli:histoprint
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/histoprint/meta.yaml:16]: This can be noarch: python

**Comment by synapticarbors** [comment]
Your windows failure is due to:

https://github.com/scikit-hep/histoprint/issues/16

**Comment by mbhall88** [comment]
> Your windows failure is due to:
> 
> [scikit-hep/histoprint#16](https://github.com/scikit-hep/histoprint/issues/16)

Ah, I didn't see that.

I guess the two options would be 
1. skipping windows build for conda
2. I think conda allows for patching. I've not done that before though and am not sure if it is able to fix this problem...

**Comment by synapticarbors** [comment]
I think patching would work. You include it in the `meta.yaml` as so:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

I haven't actually created any patches for conda recipes before, but there are a few resources out there explaining things, for example:

https://github.com/ilastik/ilastik/wiki/Creating-patches-for-Conda-recipes

**Comment by synapticarbors** [comment]
The other thing we could do is fix the issue upstream in the repo and get a release that includes this and your fix to add the `MANIFEST.in` to include the license file so that you don't have to manually package it. I'm not sure how easily it is to get the project owner to spin out a new release though.

**Comment by mbhall88** [comment]
Seems as though this PR failed on travis when it was merged. Might just be a case of restarting the build looking at the log message?

**Pull request merged**